modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/model/impl/CommerceOrderImpl.java

```diff
@@ -120,6 +148,29 @@ public int getCustomerCommerceOrderIdsCount() {
 			getCustomerCommerceOrderIdsCount(getCommerceOrderId());
 	}
 
+	@Override
+	public Folder getFolder(LocalRepository localRepository) {
+		if (localRepository == null) {
+			return null;
+		}
+
+		return localRepository.fetchFolderByExternalReferenceCode(
+			"order-" + getCommerceOrderId());
+	}
+
+	@Override
+	public LocalRepository getLocalRepository() throws PortalException {
+		Repository repository = RepositoryLocalServiceUtil.fetchRepository(
+			getGroupId(), CPConstants.SERVICE_NAME_PRODUCT);
```

Comment:
I think we should add a name for the ORDER and not use the Product one, please check with other example to determine the pattern

Reply:
Used CommerceConstants.SERVICE_NAME_COMMERCE_ORDER